backend/djangoindia/db/models/common.py

```diff
@@ -0,0 +1,26 @@
+from .base import BaseModel
+from django.db import models
+
+
+class BackgroundTaskLog(BaseModel):
+    """
+        model to hold the information about the background task that runs.
+    """
+    class StatusChoices(models.TextChoices):
+        STARTED = "started"
+        IN_PROGRESS = "in_progress"
+        FAILED = "failed"
+        COMPLETE = "complete"
+
+    name = models.CharField(max_length=256, null=False, blank=False)
+    args = models.JSONField(default=dict, null=True, blank=True)
+    kwargs = models.JSONField(default=dict, null=True, blank=True)
+    status = models.CharField(max_length=32, null=False, blank=False,default=StatusChoices.STARTED,
+                              choices=StatusChoices.choices)
+    error = models.CharField(max_length=512, null=True, blank=True)
```

Comment:
Can we change the field name from error to log? I think it will be more generic for both failed and successful tasks.